<launch>
    <arg name="namespace" default="/rs_orientation"/>
    <arg name="sensors_ns" default=""/>
    <arg name="left_sensor_topic" default="range_left"/>
    <arg name="yaw_topic" default="/print_angles/yaw"/>
    <arg name="distance_topic" default="/print_angles/distance"/>
    <arg name="right_sensor_topic" default="range_right"/>
    <arg name="map_config_file" default="$(find rs_orientation)/config/map.yaml"/>
    <arg name="pose_topic" default="pose"/>
    <group ns="$(arg namespace)">
        <param name="left_sensor_topic" value="$(arg sensors_ns)/$(arg left_sensor_topic)"/>
        <param name="right_sensor_topic" value="$(arg sensors_ns)/$(arg right_sensor_topic)"/>
        <param name="pose_topic" value="$(arg namespace)"/>
        <param name="map_config_file" value="$(arg map_config_file)"/>
        <node name="adaptive_thresholding_node" pkg="pointcloud_tests" type="adaptive_thresholding_node" output="screen"/>
    </group>
</launch>